src/drivers/tap_esc/tap_esc.cpp

```diff
@@ -262,6 +262,9 @@ TAP_ESC::init()
 		return ret;
 	}
 
+#ifdef CONFIG_ARCH_BOARD_AEROFC_V1
```

Comment:
@lucasdemarchi 

We are moving away from having Board specific #if defs in the code. 

What is the reason for this?

We can discuss on skype if need be. skype:david_s5